rust/processor/src/db/common/models/object_models/parquet_v2_objects.rs

```diff
@@ -0,0 +1,183 @@
+        let address = standardize_address(&write_resource.address.to_string());
+        if let Some(object_aggregated_metadata) = object_metadata_mapping.get(&address) {
+            // do something
+            let object_with_metadata = object_aggregated_metadata.object.clone();
+            let object_core = object_with_metadata.object_core;
+
+            let untransferrable = if object_aggregated_metadata.untransferable.as_ref().is_some() {
+                true
+            } else {
+                !object_core.allow_ungated_transfer
+            };
+            Ok(Some((
+                Self {
+                    txn_version,
+                    write_set_change_index,
+                    object_address: address.clone(),
+                    owner_address: object_core.get_owner_address(),
+                    state_key_hash: object_with_metadata.state_key_hash.clone(),
+                    guid_creation_num: object_core.guid_creation_num.clone().to_u64().unwrap(),
+                    allow_ungated_transfer: object_core.allow_ungated_transfer,
+                    is_deleted: false,
+                    untransferrable,
+                    block_timestamp,
+                },
+                CurrentObject {
+                    object_address: address,
+                    owner_address: object_core.get_owner_address(),
+                    state_key_hash: object_with_metadata.state_key_hash,
+                    allow_ungated_transfer: object_core.allow_ungated_transfer,
+                    last_guid_creation_num: object_core.guid_creation_num.clone(),
+                    last_transaction_version: txn_version,
+                    is_deleted: false,
+                    untransferrable,
+                },
+            )))
+        } else {
+            Ok(None)
+        }
+    }
+
+    /// This handles the case where the entire object is deleted
+    /// TODO: We need to detect if an object is only partially deleted
+    /// using KV store
+    pub async fn from_delete_resource(
+        delete_resource: &DeleteResource,
+        txn_version: i64,
+        write_set_change_index: i64,
+        object_mapping: &AHashMap<CurrentObjectPK, CurrentObject>,
+        conn: &mut DbPoolConnection<'_>,
+        query_retries: u32,
+        query_retry_delay_ms: u64,
+        block_timestamp: chrono::NaiveDateTime,
+    ) -> anyhow::Result<Option<(Self, CurrentObject)>> {
+        if delete_resource.type_str == "0x1::object::ObjectGroup" {
+            let resource = MoveResource::from_delete_resource(
+                delete_resource,
+                0, // Placeholder, this isn't used anyway
+                txn_version,
+                0, // Placeholder, this isn't used anyway
+            );
+            let previous_object = if let Some(object) = object_mapping.get(&resource.address) {
+                object.clone()
+            } else {
+                match DefaultObject::get_current_object(
```

Comment:
This tries to do a DB lookup to get prev owner, if it doesn't find the object core in the transaction. We probably don't want to keep this lookup for the Parquet processor. 

I think we're only migrating the `objects` table right? So we could just make the owner (and other columns) null. 
cc @ying-w